Azure/sonic-buildimage
# [action] [PR:16572] [Nokia][sonic-platform] Update Nokia sonic-platform submodule - SFP s…

This fixes https://github.com/Nokia-ION/ndk/issues/22
**Note** that this PR **must** be coupled with NDK version >= 22.9.13

#### Why I did it

To provide proper support for CMIS compliant transceiver module CDB operations (including FW related operations).



#### How I did it

Enhanced the transport subsystem so as to provide for up to 2k bytes of data to be passed to/from modules (as contrasted with the prior max of 128 bytes). 

#### How to verify it

Ensure that new FW (firmware) can be programmed to CMIS compliant module(s) using the 'sfputil firmware ...' commands.

admin@ixre-egl-board29:~$ sudo sfputil firmware download Ethernet112 /tmp/560-0087-28_Rev_61.22.21_dp04qsdd_dp04sfp8.ackit
CDB: Starting firmware download
Downloading ... [###################################-] 99% 00:00:00
CDB: firmware download complete
Firmware download complete success
Total download Time: 0:03:53.455740

admin@ixre-egl-board29:~$ sudo sfputil firmware download Ethernet112 /tmp/560-0087-23_Rev_61.20.13_dp04qsdd_dp04sfp8.ackit
CDB: Starting firmware download
Downloading ... [###################################-] 99% 00:00:00
CDB: firmware download complete
Firmware download complete success
Total download Time: 0:03:51.552884

#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)


 Tested with branch 202205

#### Description for the changelog

Properly support CDB operations for CMIS transceiver modules
**Note** that this PR **must** be coupled with NDK version >= 22.9.13
